extensions/winrt_smbus/winrt_smbus.py

```diff
@@ -0,0 +1,264 @@
+import _wini2c as i2c
+
+class SMBus:
```

Comment:
Here's the spec for smbus - http://www.smbus.org/specs/.  They are similar, but not same.  It's more like SMBus is built on top of i2c.